app/forms/floating-ip-create.tsx

```diff
@@ -123,7 +123,8 @@ export function CreateFloatingIpSideModalForm() {
             placeholder="Select pool"
           />
           <TextField
-            name="address"
+            name="ip"
+            label="Address"
```

Comment:
This hasn’t gone out to any real users, so we should just name it whatever is best. Even if it had I think I’d say the same thing. That may still be “Address”, but consistency wouldn’t be the reason.

Reply:
I think `IP address` works well and makes it clear (and maybe more clear than just `IP`); will switch to that, but happy to update again if we want it to map precisely to the API.